ext/afform/core/ang/af/afField.component.js

```diff
@@ -142,8 +142,33 @@
         });
       };
 
+      // correct the type for the value, make sure numbers are numbers and not string
+      function correctValueType(value, dataType) {
+        // let's skip type correction for null values
+        if (value == null) {
```

Comment:
Wrong operator for checking `null`, have to use `===` (also our stylechecker wants `NULL` not `null`). How about just do this instead:
```suggestion
        if (is_null(value)) {
```